volume_scroller@francislavoie.github.io/metadata.json

```diff
@@ -3,13 +3,7 @@
   "original-author": "francislavoie",
   "description": "Scroll up or down in the Top Bar to adjust volume.",
   "shell-version": [
-    "3.36",
-    "3.38",
-    "40",
-    "41",
-    "42",
-    "43",
-    "44"
+    "45"
   ],
```

Comment:
Ah, apparently they've switched to ESM. https://blogs.gnome.org/shell-dev/2023/09/02/extensions-in-gnome-45/ News to me. I don't use Gnome 45 because I'm on Pop OS, so I'm still on Gnome 42 I think.